libnmstate/nm/team.py

```diff
@@ -17,14 +17,81 @@
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 #
 
-from libnmstate.error import NmstateNotImplementedError
+import json
+
+from libnmstate.nm import connection as nm_connection
+from libnmstate.nm import nmclient
+from libnmstate.schema import Interface
 from libnmstate.schema import Team
 
 
-def create_setting(iface_state):
-    setting = None
+def create_setting(iface_state, base_con_profile):
+    team_setting = None
     team_config = iface_state.get(Team.CONFIG_SUBTREE)
-    if team_config:
-        raise NmstateNotImplementedError
+    if base_con_profile:
```

Comment:
This means that you can never create a new team, you can just edit an existing one.
Or do I miss something here?

Reply:
Oops. I missed to amend some changes into the commit. Fixed :-)